arkouda/pdarraycreation.py

```diff
@@ -15,15 +15,14 @@
     bigint,
 )
 from arkouda.dtypes import dtype as akdtype
+from arkouda.dtypes import float64, get_byteorder, get_server_byteorder
```

Comment:
If these are not being aliased, is there a reason they were moved from the import 2 lines below?

Reply:
If that is what `isort` is doing let's leave it. Just wanted to check there wasn't some other reason.